packages/react/src/ResourceArrayInput/ResourceArrayInput.tsx

```diff
@@ -101,6 +100,7 @@ export function ResourceArrayInput(props: ResourceArrayInputProps): JSX.Element
                 property={props.property}
                 name={props.name + '.' + valueIndex}
                 path={props.path}
+                indexedPath={getIndexedPath(props.path, props.indexedPath, valueIndex)}
```

Comment:
Can you help me build intuition here regarding `path` vs `indexedPath`?  I assume `indexedPath` has something to do with slices?

Reply:
UPDATE: I went through and replace all the naming from `indexedPath` to `expression`.